bin/commands/init/mvs/index.sh

```diff
@@ -11,104 +10,16 @@
 # Copyright Contributors to the Zowe Project.
 #######################################################################
 
-print_level1_message "Initialize Zowe custom data sets"
-
-###############################
-# constants
-cust_ds_list="parmlib|Zowe parameter library|dsntype(library) dsorg(po) recfm(f b) lrecl(80) unit(sysallda) space(15,15) tracks
-jcllib|Zowe JCL library|dsntype(library) dsorg(po) recfm(f b) lrecl(80) unit(sysallda) space(15,15) tracks
-authLoadlib|Zowe authorized load library|dsntype(library) dsorg(po) recfm(u) lrecl(0) blksize(32760) unit(sysallda) space(30,15) tracks
-authPluginLib|Zowe authorized plugin library|dsntype(library) dsorg(po) recfm(u) lrecl(0) blksize(32760) unit(sysallda) space(30,15) tracks"
+init_missing_yaml_properties
```

Comment:
1. The command `zwe init mvs -c zowe.with.useConfigmgr.false.yaml` returns:
```
These configurations need to be added to your YAML configuration file:

node:
  home: "/node/v18r0/IBM/node-v18.19.0-os390-s390x-202512322561"
java:
  home: "/java/java64bt/v8r0m0/usr/lpp/java/J8.0_64"

>> Please manually update "./zowe.with.useConfigmgr.false.yaml" before you start Zowe.

Error ZWEL0316E: Command requires zowe.useConfigmgr=true to use.
```

The output is confusing. Other `zwe init subcommands` are returning only `Error ZWEL0316E: Command requires zowe.useConfigmgr=true to use.`

2. Why is the `init_missing_yaml_properties` here and in [`zwe init`](https://github.com/zowe/zowe-install-packaging/blob/319bacbde332c0d94afdde166aa116c8f63d6a23/bin/commands/init/index.sh#L14)? Is this forgotten line?